core/tests/protractor_desktop/contributorDashboard.js

```diff
@@ -65,6 +65,11 @@ describe('Contributor dashboard page', function() {
     await users.createUser(USER_EMAILS[0], 'user0');
     await users.createUser(USER_EMAILS[1], 'user1');
     await users.createAndLoginAdminUser(ADMIN_EMAIL, 'management');
+    await adminPage.editConfigProperty(
+      'Whether the contributor can suggest questions for skill opportunities.',
+      'Boolean', async function(elem) {
+        await elem.setValue(true);
```

Comment:
Before setting the element's value, can you check that it has loaded with a `waitFor` call?